lib/rucio/common/utils.py

```diff
@@ -560,7 +560,7 @@ class ScopeExtractionAlgorithms(PolicyPackageAlgorithms):
     Handle scope extraction algorithms
     """
 
-    _algorithm_type = 'scope'
+    _algorithm_type: 'POLICY_ALGORITHM_TYPES_LITERAL' = 'scope'
```

Comment:
Quick question about the type annotations - I see we're using quotes around `POLICY_ALGORITHM_TYPES_LITERAL` in some places. Since it's imported from constants, would it be cleaner to remove the quotes for better type checking? Just wondering if there was a specific reason for keeping them quoted.

Also, would it make sense to consider consistent variable naming here? For instance, using `algorithm_type` instead of `_algorithm_type` might follow more conventional naming patterns. 

```suggestion
    algorithm_type: POLICY_ALGORITHM_TYPES_LITERAL = 'scope'
```

Reply:
Removing the quotes is a good suggestion so I've done that. The leading underscore on the variable name has always been there, I think it's intended to signify a variable used internally by the class, but I don't have strong views on whether this should be kept.

Reply:
Thanks for addressing the findings.

You're right. Though Python doesn't enforce encapsulation, the underscore marks this as a protected member by convention - something subclasses can use but external code shouldn't access directly. Removing it would break this widely-followed convention and create inconsistency with your existing codebase. Let's keep it as is - not actually an issue that needs fixing.